test/helpers/database-setup-for-tests.js

```diff
@@ -270,7 +271,7 @@ function deleteByReference (schemaTable, reference) {
 }
 
 module.exports.deleteAll = function (reference) {
-  return deleteByReference('IntSchema.Task', reference)
+  return deleteByReference('IntSchema.Task')
```

Comment:
Is there a reason the reference parameter was removed?  From what I can see, with the parameter removed, this will leave any newly created tasks in the database.